src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/RunnableProjectConfig.cs

```diff
@@ -664,8 +664,11 @@ private IReadOnlyList<IReplacementTokens> ProduceSymbolFilenameReplacements()
                         if (_configuration.Forms.TryGetValue(formName, out IValueForm valueForm))
                         {
                             string symbolName = symbol.Name + "{-VALUE-FORMS-}" + formName;
-                            string processedFileReplacement = valueForm.Process(_configuration.Forms, p.FileRename);
-                            GenerateFileReplacementsForSymbol(processedFileReplacement, symbolName, filenameReplacements);
+                            string? processedFileReplacement = valueForm.Process( p.FileRename, _configuration.Forms);
```

Comment:
nit: space after `(`. Interesting we do not have analyzer rule for this.

Reply:
Enabled IDE0055 for this